frontend/router/components/AuthAnyMaintainerAdminObserverPlusRoutes/AuthAnyMaintainerAdminObserverPlusRoutes.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from "react";
+import { useErrorHandler } from "react-error-boundary";
+import { AppContext } from "context/app";
+
+interface IAuthAnyMaintainerAdminObserverPlusRoutesProps {
+  children: JSX.Element;
+}
+
+/**
+ * Checks if a user is any maintainer, admin, or observer plus when routing
+ */
+const AuthAnyMaintainerAdminObserverPlusRoutes = ({
+  children,
+}: IAuthAnyMaintainerAdminObserverPlusRoutesProps) => {
+  const handlePageError = useErrorHandler();
+  const {
+    currentUser,
+    isGlobalAdmin,
+    isGlobalMaintainer,
+    isAnyTeamAdmin,
+    isAnyTeamMaintainer,
+    isObserverPlus,
```

Comment:
isObserverPlus encompasses the team observer plus

Reply:
Fixed! Issue was if the team wasn't select that they were observer plus for, it would block it, so now I added isAnyTeamObserverPlus to not block the route